cmd/podman/runlabel.go

```diff
@@ -94,6 +94,14 @@ func runlabelCmd(c *cli.Context) error {
 		newImage       *image.Image
 	)
 
+	// Evil images could trick into recursively executing the runlabel
+	// command.  Avoid this by setting the "PODMAN_RUNLABEL_NOEXEC" env
+	// variable when executing a label first.
+	noexec := os.Getenv("PODMAN_RUNLABEL_NOEXEC")
+	if noexec == "1" {
+		return fmt.Errorf("runlabels cannot execute the runlabel command")
```

Comment:
Maybe reword this to "cannot execute runlabel because noexec environment variable has been set?

Reply:
I don't like this one as it doesn't guide the user to resolve the issue. I am also not very happy about the proposed error message but it couldn't find a "better" one that tells the user that recursion isn't allowed.